### PR TITLE
update documentation to match FortiOS device tracker updates

### PR DESCRIPTION
**Pull request by kimfrellsen**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by kimfrellsen** [comment]
This is documentation to PR https://github.com/home-assistant/core/pull/61970

File: source/_integrations/fortios.markdown [hunk 1]
```diff
@@ -5,17 +5,19 @@ ha_category:
   - Presence Detection
 ha_release: 0.97
 ha_iot_class: Local Polling
+ha_quality_scale: silver
```
Comment (MartinHjelmare) [inline source/_integrations/fortios.markdown]: Changing the quality scale requires updating the manifest.json in the core integration first.

**Review by frenck — CHANGES_REQUESTED** [review]
Please make sure the CI/builds pass

File: source/_integrations/fortios.markdown (hunk 1 above)
Comment (kimfrellsen) [reply source/_integrations/fortios.markdown]: Thanks. I have removed the quality_scale. It is not of importance to me.
I do not understand the Netlify errors. They are pointing towards the "ha_iot_class: Local Polling" line and complaining of headers. I have tried to encapsulate them in " without any luck.

File: source/_integrations/fortios.markdown [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
 ---
 
-This integration enables Home Assistant to do device tracking of devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).
+The FortiOS presence detection enables Home Assistant to track devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).
 
-The integration relies on the [fortiosapi](https://pypi.org/project/fortiosapi/).
-The integration has been tested both on FortiGate appliance and FortiGate VM running SW FortiOS v. 6.0.x and 6.2.0.
+The integration relies on the [fortiosapi](https://pypi.org/project/fortiosapi/).  
+The integration has been verified on FortiGate appliances and FortiGate VM running FortiOS v. 6.4.x (up to 6.4.8) and 7.0.x (up to 7.0.4).  
+The latest update to the integration use this API: ```<host>:<port>/api/v2/monitor/user/device/query``` which is only available in FortiOS 6.4.3 and later. So minimum supported version is on FortiOS 6.4.3.
```
Comment (MartinHjelmare) [inline source/_integrations/fortios.markdown]:
```suggestion
The integration uses this API: ```<host>:<port>/api/v2/monitor/user/device/query``` which is only available in FortiOS 6.4.3 and later. So minimum supported version is on FortiOS 6.4.3.
```

File: source/_integrations/fortios.markdown [hunk 3]
```diff
@@ -31,16 +33,16 @@ device_tracker:
 
 {% configuration %}
 host:
-    description: Hostname or IP address of the FortiGate.
-    required: true
-    type: string
+  description: Hostname or IP address of the FortiGate. Optionally portname can be added like this "10.10.10.10:443". Remember qoutes if portnumber is added.
```
Comment (MartinHjelmare) [inline source/_integrations/fortios.markdown]:
```suggestion
  description: Hostname or IP address of the FortiGate. Optionally port name can be added like this "10.10.10.10:443". Remember quotes if port number is added.
```

File: source/_integrations/fortios.markdown [hunk 4]
```diff
@@ -31,16 +33,16 @@ device_tracker:
...
+  required: true
+  type: string
 token:
-    description: "See: [Generate an API token for FortiOS](https://registry.terraform.io/providers/fortinetdev/fortios/latest/docs/guides/fgt_token) for how to create an API token. Remember this integration only needs read access to a FortiGate, so configure the API user to only to have limited and read-only access."
-    required: true
-    type: string
+  description: See: [Generate an API token for FortiOS](https://docs.fortinet.com/document/forticonverter/6.2.0/online-help/866905/connect-fortigate-device-via-api-token) for how to create an API token. Remember this integration only needs read access to a FortiGate, so configure the API user to only to have limited and read-only access.
```
Comment (MartinHjelmare) [inline source/_integrations/fortios.markdown]:
```suggestion
  description: See: [Generate an API token for FortiOS](https://docs.fortinet.com/document/forticonverter/6.2.0/online-help/866905/connect-fortigate-device-via-api-token) for how to create an API token. This integration only needs read access to a FortiGate, so configure the API user to only to have limited and read-only access.
```

**Comment by frenck** [comment]
Rebased the PR to deal with the CI issues

File: source/_integrations/fortios.markdown [hunk 5]
```diff
@@ -1,28 +1,30 @@
...
-  - Presence Detection
-ha_release: 0.97
-ha_iot_class: Local Polling
-ha_codeowners:
-  - '@kimfrellsen'
+title: "FortiOS"
+description: "Instructions on how to use Fortinet FortiOS to track devices in Home Assistant."
+ha_category: "Presence Detection"
+ha_iot_class: "Local Polling"
+ha_release: "0.97"
```
Comment (frenck) [inline source/_integrations/fortios.markdown]:
```suggestion
title: FortiOS
description: Instructions on how to use Fortinet FortiOS to track devices in Home Assistant.
ha_category: Presence Detection
ha_iot_class: Local Polling
ha_release: 0.97
```

File: source/_integrations/fortios.markdown [hunk 6]
```diff
@@ -1,28 +1,30 @@
...
+  - '@kimfrellsen'
 ---
 
-This integration enables Home Assistant to do device tracking of devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).
+The FortiOS presence detection enables Home Assistant to track devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).
 
-The integration relies on the [fortiosapi](https://pypi.org/project/fortiosapi/).
-The integration has been tested both on FortiGate appliance and FortiGate VM running SW FortiOS v. 6.0.x and 6.2.0.
+The integration relies on the [fortiosapi](https://pypi.org/project/fortiosapi/).  
+The integration has been verified on FortiGate appliances and FortiGate VM running FortiOS v. 6.4.x (up to 6.4.8) and 7.0.x (up to 7.0.4).
```
Comment (frenck) [inline source/_integrations/fortios.markdown]:
```suggestion
The integration relies on the [fortiosapi](https://pypi.org/project/fortiosapi/). It has been verified on FortiGate appliances and FortiGate VM running FortiOS v. 6.4.x (up to 6.4.8) and 7.0.x (up to 7.0.4).  
```

File: source/_integrations/fortios.markdown [hunk 7]
```diff
@@ -1,28 +1,30 @@
...
 ---
 
-This integration enables Home Assistant to do device tracking of devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).
+The FortiOS presence detection enables Home Assistant to track devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).
 
-The integration relies on the [fortiosapi](https://pypi.org/project/fortiosapi/).
-The integration has been tested both on FortiGate appliance and FortiGate VM running SW FortiOS v. 6.0.x and 6.2.0.
+The integration relies on the [fortiosapi](https://pypi.org/project/fortiosapi/).  
+The integration has been verified on FortiGate appliances and FortiGate VM running FortiOS v. 6.4.x (up to 6.4.8) and 7.0.x (up to 7.0.4).  
+The integration uses this API: ```<host>:<port>/api/v2/monitor/user/device/query``` which is only available in FortiOS 6.4.3 and later. So minimum supported version is on FortiOS 6.4.3.
```
Comment (frenck) [inline source/_integrations/fortios.markdown]: This is an implementation detail. Not relevant for the average end-user.
```suggestion
The minimum version supported is FortiOS 6.4.3.
```

File: source/_integrations/fortios.markdown [hunk 8]
```diff
@@ -1,28 +1,30 @@
...
 
 All devices with a MAC address identified by FortiGate would be tracked, this covers both Ethernet and Wi-Fi devices, including devices detected by LLDP.
 
 The integration is based on the Home Assistant `device_tracker` platform.
 
+### Example Configuration:
+This example uses the FortiOS integration as device tracker, with API token and does not verify the SSL certificate.
```
Comment (frenck) [inline source/_integrations/fortios.markdown]:
```suggestion
### Example Configuration:

This example uses the FortiOS integration as a device tracker, with an API token and does not verify the SSL certificate.

```

File: source/_integrations/fortios.markdown [hunk 9]
```diff
@@ -31,16 +33,16 @@ device_tracker:
...
+  type: string
 verify_ssl:
-    description: If the SSL certificate should be verified. In most home cases users do not have a verified certificate.
-    required: false
-    type: boolean
-    default: false
+  description: If the SSL certificate should be verified. In most home cases users do not have a verified certificate.
+  required: false
+  type: boolean
+  default: false
```
Comment (frenck) [inline source/_integrations/fortios.markdown]:
```suggestion
  default: false
```

File: source/_integrations/fortios.markdown [hunk 10]
```diff
@@ -1,28 +1,31 @@
 ---
 title: FortiOS
 description: Instructions on how to use Fortinet FortiOS to track devices in Home Assistant.
-ha_category:
-  - Presence Detection
-ha_release: 0.97
+ha_category: Presence Detection
```
Comment (frenck) [inline source/_integrations/fortios.markdown]:
```suggestion
ha_category: 
 - Presence Detection
```

**Review by frenck — APPROVED** [review]
Thanks, @kimfrellsen 👍

**Pull request merged**